schemas/telemetry/configuration-schema.json

```diff
@@ -49,6 +49,11 @@
                   "minimum": 0,
                   "maximum": 100
                 },
+                "trace_context_injection": {
+                  "type": "string",
+                  "description": "The opt-in configuration to add trace context",
+                  "readOnly": false
+                },
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** what about documenting the possible values? 
FMU:‏
```suggestion
                "trace_context_injection": {
                  "type": "string",
                  "description": "The opt-in configuration to add trace context",
                  "enum": ["all", "sampled"],
                  "readOnly": false
                },
```